src/libopenrave/kinbodygeometry.cpp

```diff
@@ -1468,6 +1510,14 @@ uint8_t KinBody::GeometryInfo::GetSideWallExists() const
     return mask;
 }
 
+dReal KinBody::GeometryInfo::GetCylinderRadius() const
+{
+    if (_vGeomData.z == 0 || _vGeomData.x != _vGeomData.z) {
```

Comment:
@superfashi Shouldn't this condition be `_vGeomData.z != 0 && _vGeomData.x != _vGeomData.z` instead?

Reply:
You are right. Can you send a PR? I am currently on something else. Thank you.